README.md

```diff
@@ -370,13 +370,30 @@ Setting up a local dev environment is simple, thanks to the rust ecosystem:
 - Install [`rustup` & `cargo`](https://doc.rust-lang.org/cargo/getting-started/installation.html).
 - That's it! Running `cargo test` should complete successfully.
 - For more advanced development; e.g. adjusting `insta` outputs or compiling for
-  web, run the commands in [Taskfile.yml](Taskfile.yml), either with copy &
-  paste or by installing [Task](https://taskfile.dev/#/installation) and running
-  `task install-dev-tools`.
+  web, run the commands in [Taskfile.yml](Taskfile.yml), either by copying &
+  pasting or by installing [Task](https://taskfile.dev/#/installation) and
+  running `task install-dev-tools`.
 - For quick contributions, hit `.` in GitHub to launch a [github.dev
   instance](https://github.dev/max-sixty/prql).
 - Any problems: post an issue and we'll help.
 
+### Contributors
+
+Many thanks to those who've made our progress possible:
+
+[![Contributors](https://contrib.rocks/image?repo=max-sixty/prql)](https://github.com/max-sixty/prql/graphs/contributors)
+
+### Core developers
+
+We have a few core developers who are responsible for reviewing code, making
+decisions on the direction of the language, and project administration:
+
+- @aljazerzen
```

Comment:
You've put me on the first place, what an honor :smile:

Reply:
How do I get to first place ?

Reply:
It's alphabetical! 

But actually it should be in reverse alphabetical, to correct for the privilege of those with higher ranking letters.